### DISCUSSION TITLE
How to retrieve selected set name or number from Rshiny interactiveChart()

### DISCUSSION THREAD

**Discussion by baeywen** [upsetjs/upsetjs#83] [Q&A] [answered]
The example Rshiny code looks great in https://upset.js.org/integrations/r/index.html
But i am trying to interactively retrieve more information after users click/highlight a selected column. so that i can plot more figures based on user's selection. is there any way to retrieve this information in Rshiny?

**Comment by sgratzl** [comment, marked as answer]
see also https://github.com/upsetjs/upsetjs_r/blob/main/r_package/shiny/events.R

**Reply by baeywen** [reply]
really appreciate it
